Upstream-Status: Pending

coreutils: fix for dummy-man usage

The options should be before the final argument, otherwise, the following error
would appear when compiling.

"dummy-man: too many non-option arguments"

Signed-off-by: Chen Qi <Qi.Chen@windriver.com>
---
 man/local.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/local.mk b/man/local.mk
index ba8fbf7..3833458 100644
--- a/man/local.mk
+++ b/man/local.mk
@@ -101,8 +101,8 @@ man/dynamic-deps.mk: Makefile
 	  && $(run_help2man)						\
 		     --source='$(PACKAGE_STRING)'			\
 		     --include=$(srcdir)/man/$$name.x			\
-		     --output=$$t/$$name.1 $$t/$$argv			\
 		     --info-page='coreutils \(aq'$$name' invocation\(aq' \
+		     --output=$$t/$$name.1 $$t/$$argv			\
 	  && sed \
 	       -e 's|$*\.td/||g' \
 	       -e '/For complete documentation/d' \
-- 
1.9.1

